### PR TITLE
Support more than one header in External Channel

### PR DESCRIPTION
**Pull request by jcbalmeida**
Solves the issue https://github.com/rapidpro/rapidpro/issues/1547

Obs.: It need https://github.com/nyaruka/courier/pull/386 to work

### CODE REVIEW

**Comment by matmsa27** [comment]
Hi @rowanseymour , any news on this and the courier PR? Thanks

**Comment by rowanseymour** [comment]
@matmsa27 just want small tweak re https://github.com/nyaruka/courier/pull/386#pullrequestreview-795177617 as we'll be deploying the courier side before we deploy this RP bit

**Comment by matmsa27** [comment]
Nice, thanks @rowanseymour , I saw here that courier was merged. Please, any adjust here that you want on this PR, we will be available to fix ASAP.

File: temba/channels/migrations/0136_external_headers.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.8 on 2021-11-01 22:37
+
+from django.db import migrations
+
+
+def foward(apps, schema_editor):  # pragma: no cover
```
Comment (rowanseymour) [inline temba/channels/migrations/0136_external_headers.py:6]: Ok so our general rules for data migrations are..

1. Just print a summary at the end like `123 channels updated, 5 ignored`
2. Write a unit test using `MigrationTest` (e.g. see https://github.com/nyaruka/rapidpro/blob/ecc9bc2808673617116d03bf7690cc52d8410c05/temba/msgs/tests.py#L2847)
3. Don't worry about reverse-ability

**Review by rowanseymour — CHANGES_REQUESTED** [review]

**Pull request closed without merging**